src/SFA.DAS.ApprenticeCommitments/InnerApi/Requests/CreateApprenticeshipRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+
+namespace SFA.DAS.ApprenticeCommitments.InnerApi.Requests
+{
+    public class CreateApprenticeshipRequest : IPostApiRequest
+    {
+        public string PostUrl => "/apprenticeships";
+
+        public object Data { get; set; }
+    }
+
+    public class CreateApprenticeshipRequestData
+    {
+        public Guid RequestId { get; set; }
```

Comment:
This is called RegistrationId in the inner API, not sure which name is right

Reply:
I *thought* I had copied that from the inner API, obviously not 😆